components/form-builder/app/Settings.tsx

```diff
@@ -69,6 +70,22 @@ const HintText = ({ id, children }: { id: string; children?: JSX.Element | strin
   );
 };
 
+const InvalidEmailError = ({ isActive }: { isActive: boolean }) => {
+  const { t } = useTranslation("form-builder");
+
+  return isActive ? (
+    <div className="mt-2 mb-2" role="alert">
+      <span className="text-red text-sm font-bold">{t("settingsInvalidEmailAlertTitle")}</span>
```

Comment:
This should probably be a heading. I don't think there are any other headings on the page, so could be H2 and just use tailwind classes to size it appropriately.